contracts/commands/BaseAAveFlashLoanCommand.sol

```diff
@@ -56,12 +57,14 @@ abstract contract BaseAAveFlashLoanCommand is ICommand, IFlashLoanReceiver {
     constructor(
         IServiceRegistry _serviceRegistry,
         ILendingPool _lendingPool,
-        AaveProxyActions _aaveProxyActions
+        AaveProxyActions _aaveProxyActions,
+        address _WETH
     ) {
         aaveProxyActions = _aaveProxyActions;
         serviceRegistry = _serviceRegistry;
         lendingPool = _lendingPool;
         self = address(this);
+        WETH = _WETH;
```

Comment:
weth can be added to service registry instead and fetched from there, maybe it is there already